lib/SILGen/SILGenStmt.cpp

```diff
@@ -277,6 +277,22 @@ Condition SILGenFunction::emitCondition(SILValue V, SILLocation Loc,
   return Condition(TrueBB, FalseBB, ContBB, Loc);
 }
 
+static void getExpandedElements(BraceStmt *stmt,
+                                SmallVectorImpl<ASTNode> &results) {
+  llvm::function_ref<void(ASTNode)> recurse;
+  recurse = [&](ASTNode node) {
+    if (auto *decl = node.dyn_cast<Decl *>()) {
+      if (auto *med = dyn_cast<MacroExpansionDecl>(decl)) {
+        for (auto expandedNode : med->getRewritten())
+          recurse(expandedNode);
+        return;
+      }
+    }
+    results.push_back(node);
```

Comment:
Lots of separate recursion implementations for visiting expansions.

Reply:
Eliminated this one.